src/main/sensors/acceleration.c

```diff
@@ -485,7 +491,11 @@ void accUpdate(timeUs_t currentTimeUs, rollAndPitchTrims_t *rollAndPitchTrims)
         }
     }
 
-    alignSensors(acc.accADC, acc.dev.accAlign);
+    if (acc.dev.accAlign == ALIGN_CUSTOM) {
```

Comment:
Personally I'd prefer to use matrix in all cases ...

Reply:
me too, I found the code so much cleaner in the previous PR.

Reply:
I really do not see any point in forcing 99.9% of users to use a gyro loop that requires a lot more calculation for no benefit whatsoever, just to support some exotic hardware that does not provide any benefit.

Reply:
ok, we'll carry on with the ALIGN_CUSTOM as planned.